aspnetcore/blazor/common/samples/3.x/BlazorSample/Components/HeadingComponent.razor

```diff
@@ -21,7 +21,7 @@
             _italicsCheck field.
         *@
         <input type="checkbox" id="italicsCheck" 
-               bind="@_italicsCheck" />
+               @bind="@_italicsCheck" />
```

Comment:
note that this can also be simplified to `@bind="@_italicsCheck"`. Having extra `@` there seems wierd now that the directive has one.